sdk/src/wallet/migration/chrysalis.rs

```diff
@@ -58,8 +58,12 @@ pub(crate) struct AccountDetailsDto {
     native_token_foundries: HashMap<String, Value>,
 }
 
-pub async fn migrate_db_chrysalis_to_stardust(storage_path: String, password: Option<Password>) -> Result<()> {
-    let storage_path = Path::new(&storage_path);
+pub async fn migrate_db_chrysalis_to_stardust(
+    storage_path: impl Into<String> + Send,
```

Comment:
`impl Into<PathBuf>` could also work probably

Reply:
We doing this or no?

Reply:
Yes, updated